doc/langdef.md

```diff
@@ -308,16 +308,15 @@ Escape sequences are a backslash (`\ `) followed by one of the following:
     *   `t`: horizontal tab
     *   `v`: vertical tab
 *   A `u` followed by four hexadecimal characters, encoding a Unicode code point
-    in the
-    [BMP](https://en.wikipedia.org/wiki/Plane_\(Unicode\)#Basic_Multilingual_Plane).
-    Characters in other Unicode planes can be represented with surrogate pairs.
-    Valid only for string literals.
```

Comment:
I can't find any evidence of support for surrogate pairs in the `cel-go` or `cel-java` (I can't get `cel-cpp` to build). And really, why would they be supported since:
- they are not a not a UTF-8 construct
- they cannot express anything that isn't more explicitly expressed via a `\U`-prefixed 32-bit code point